src/components/404.vue

```diff
@@ -1,373 +1,179 @@
-        fill="black"
-      />
-      <path
-        d="M73.1301 45V39.6798C73.9725 40.7611 75.1174 41.2802 76.3486 41.2802C79.0272 41.2802 80.9713 39.2689 80.9713 35.787C80.9713 32.3051 79.0272 30.3154 76.3486 30.3154C75.0742 30.3154 73.9293 30.8777 73.1301 31.8941V30.5749H70.3868V45H73.1301ZM78.1199 35.787C78.1199 37.6036 77.0615 38.8364 75.4846 38.8364C74.599 38.8364 73.5837 38.2957 73.1301 37.6469V33.9487C73.5837 33.2783 74.599 32.7592 75.4846 32.7592C77.0615 32.7592 78.1199 33.992 78.1199 35.787Z"
-        fill="black"
-      />
-      <path d="M83.0225 41.0207H85.7659V26.5956H83.0225V41.0207Z" fill="black" />
-      <path
-        d="M94.7504 41.0207H97.4937V34.2731C97.4937 31.267 95.312 30.3154 92.9359 30.3154C91.2942 30.3154 89.6525 30.8344 88.3781 31.959L89.4149 33.7973C90.3005 32.9755 91.3374 32.5646 92.4607 32.5646C93.8431 32.5646 94.7504 33.2566 94.7504 34.3164V35.7654C94.0591 34.9219 92.8279 34.4894 91.4454 34.4894C89.7821 34.4894 87.8164 35.3761 87.8164 37.8199C87.8164 40.1556 89.7821 41.2802 91.4454 41.2802C92.8063 41.2802 94.0375 40.7828 94.7504 39.9393V41.0207ZM94.7504 38.512C94.2967 39.1175 93.4327 39.4203 92.5471 39.4203C91.467 39.4203 90.5814 38.858 90.5814 37.8848C90.5814 36.8683 91.467 36.306 92.5471 36.306C93.4327 36.306 94.2967 36.6088 94.7504 37.2143V38.512Z"
-        fill="black"
-      />
-      <path
-        d="M107.257 41.0207H110V33.6459C110 31.613 108.898 30.3154 106.609 30.3154C104.902 30.3154 103.628 31.1372 102.958 31.9374V30.5749H100.215V41.0207H102.958V33.992C103.412 33.3648 104.254 32.7592 105.334 32.7592C106.501 32.7592 107.257 33.2566 107.257 34.7056V41.0207Z"
-        fill="black"
-      />
-    </svg>
-    <div id="logo">
-      <div id="first_logo_box" class="color_box"></div>
-      <div id="second_logo_box" class="color_box"></div>
-      <div id="third_logo_box" class="color_box"></div>
-      <div id="fourth_logo_box" class="color_box"></div>
+<div>
+  <div class = "body-container">
+    <!--TOP BAR-->
+    <div class="container p-0">
+        <div class="row top-bar justify-content-between no-gutters">
+            <div class="col-10">
+                <img class="logo" src="@/assets/images/logo.svg" alt = "logo" />
+              </div>
+        </div>
     </div>
-    <div id="band">
-      <div id="first_four">
-        <div id="first_box1" class="color_box"></div>
-        <div id="second_box1" class="color_box"></div>
-        <div id="third_box1" class="color_box"></div>
-        <div id="fourth_box1" class="color_box"></div>
+    <!-- <img data-v-ef68022e="" src="img/logo.4d084161.svg" alt="logo" class="logo"> -->
```

Comment:
Please remove the commented out code here as well, and further in the file like in the CSS.